pkg/mappings/resources/volumesnapshotclasses.go

```diff
@@ -13,7 +13,7 @@ import (
 var volumeSnapshotClassesCRD string
 
 func CreateVolumeSnapshotClassesMapper(ctx *synccontext.RegisterContext) (synccontext.Mapper, error) {
-	if !ctx.Config.Sync.ToHost.VolumeSnapshots.Enabled {
+	if !ctx.Config.Sync.FromHost.VolumeSnapshotClasses.Enabled {
```

Comment:
I'm not understanding why we check this?   Since it tries to return a NewMirrorMapper whether its enabled or not, if the EnsureCRD check is successful.  Is it to get the error from EnsureCRD instead of from `apiutil.GVKForObject(obj, scheme.Scheme)` used in NewMirrorMapper?

Reply:
The `sync.toHost.volumesnapShot` toggle was split into three different toggles in a previous pull request.

This change is mostly for consistency's sake so that the volumesnapshotclass mapper is controlled by its toggle.